SPECS/systemd/systemd.spec

```diff
@@ -1,62 +1,342 @@
+%bcond zstd   1
+
+# Bootstrap may be needed to break circular dependencies with cryptsetup,
+# e.g. when re-building cryptsetup on a json-c SONAME-bump.
+%bcond bootstrap 0
+%bcond tests     1
+%bcond lto       1
+
+# Support for quick builds with rpmbuild --build-in-place.
+# See README.build-in-place.
+%bcond inplace 0
+
 Name:           systemd
-Version:        254.5
-Release:        1%{?dist}
-License:        LGPLv2+ AND GPLv2+ AND MIT
-Vendor:         Microsoft Corporation
-Distribution:   Mariner
-Group:          System Environment/Security
-URL:            https://www.freedesktop.org/wiki/Software/systemd/
-Source0:        https://github.com/%{name}/%{name}-stable/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
-Source1:        95-disable-systemd-oomd.preset
-BuildRequires:  audit-devel
-BuildRequires:  cryptsetup-devel
-BuildRequires:  docbook-dtd-xml
-BuildRequires:  docbook-style-xsl
-BuildRequires:  gettext
-BuildRequires:  glib-devel
-BuildRequires:  gperf
-BuildRequires:  intltool
-BuildRequires:  kbd
-BuildRequires:  kmod-devel
+Url:            https://systemd.io
+%if %{without inplace}
+Version:        255
+%else
+# determine the build information from local checkout
+Version:        %(tools/meson-vcs-tag.sh . error | sed -r 's/-([0-9])/.^\1/; s/-g/_g/')
+%endif
+Release:        2%{?dist}
+
+# FIXME - hardcode to 'stable' for now as that's what we have in our blobstore
+%global stable 1
+#global stable %(c="%version"; [ "$c" = "${c#*.*}" ]; echo $?)
+
+# For a breakdown of the licensing, see README
+License:        LGPL-2.1-or-later AND MIT AND GPL-2.0-or-later
+Summary:        System and Service Manager
+
+# download tarballs with "spectool -g systemd.spec"
+%if %{defined commit}
+Source0:        https://github.com/systemd/systemd%{?stable:-stable}/archive/%{commit}/%{name}-%{shortcommit}.tar.gz
+%else
+%if 0%{?stable}
+Source0:        https://github.com/systemd/systemd-stable/archive/v%{version_no_tilde}/%{name}-%{version_no_tilde}.tar.gz
+%else
+Source0:        https://github.com/systemd/systemd/archive/v%{version_no_tilde}/%{name}-%{version_no_tilde}.tar.gz
+%endif
+%endif
+# This file must be available before %%prep.
+# It is generated during systemd build and can be found in build/src/core/.
+Source1:        triggers.systemd
+Source2:        split-files.py
+
+# Prevent accidental removal of the systemd package
+Source4:        yum-protect-systemd.conf
+
+Source6:        sysctl.conf.README
+Source7:        systemd-journal-remote.xml
+Source8:        systemd-journal-gatewayd.xml
+Source9:        20-yama-ptrace.conf
+Source10:       systemd-udev-trigger-no-reload.conf
+
+Source14:       10-oomd-defaults.conf
+Source15:       10-oomd-per-slice-defaults.conf
+Source16:       10-timeout-abort.conf
+Source17:       10-map-count.conf
+
+Source21:       macros.sysusers
+Source22:       sysusers.attr
+Source23:       sysusers.prov
+Source24:       sysusers.generate-pre.sh
+
+Source25:       98-default-mac-none.link
```

Comment:
Why all the new config files? Was the 254->255 version bump a significant change? Or did you just want to make our configuration explicit while you were updating this package (+1 to that)?

Reply:
direct from fedora

Reply:
Could you elaborate? We don't need to keep other distro-specific bits in our specs.

Reply:
please take a look at the entire patch series